webpack.config.js

```diff
@@ -58,7 +58,7 @@ module.exports = {
     watchOptions: {
         aggregateTimeout: 500,
         poll: 1000,
-        ignored: /node_modules/,
+        ignored: "**/{node_modules,dist}/**"
```

Comment:
I think this would be a bit more readable like this:

```suggestion
        ignored: [
            "**/node_modules/**",
            "**/dist/**",
        ],
```

Reply:
This makes sense to me.